Descomentado as partes erradas, RealizadoServico

LucasPerezAlves opened the pull request:
-Foi implementado o AdministradorServico
-Foi descomentado as linhas do AdministradorRepositorio e IAdministradorRepositorio
-Foi implementado o AdministradorCadastrarViewModel

LucasPerezAlves committed:
Descomentado as partes erradas, RealizadoServico
-Foi implementado o AdministradorServico
-Foi descomentado as linhas do AdministradorRepositorio e IAdministradorRepositorio
-Foi implementado o AdministradorCadastrarViewModel

LucasPerezAlves committed:
merge
-Resolvi os conflitos

GuilhermeDMueller reviewed (commented):
Remover Linhas em branco

LucasPerezAlves committed:
AdministradorEditarViewModel/AdministradorServico
-Foi realizado o AdministradorEditarViewModel
-Terminado o AdministradorServico por completo
